schemas/mod-source-record-manager/deleteJobExecutionsRespDto.json

```diff
@@ -0,0 +1,16 @@
+{
```

Comment:
We already have word 'response' in name, so I think dto its not needed in naming.
We can name objects like (deleteJobExecutionsReq, deleteJobExecutionsResp) or (deleteJobExecutionsReqest, deleteJobExecutionsResponse)

Reply:
Hi @SerhiiNosko - PR updated as per suggestions, Thanks 🙂